src/sources/kubernetes/test.rs

```diff
@@ -112,7 +112,7 @@ spec:
         emptyDir: {}
       containers:
       - name: vector
-        image: ktff/vector-kube-watch-client:latest
+        image: ktff/vector-kube-newline-bug:latest
```

Comment:
Is this intentional?

Reply:
Yes. [This](https://github.com/timberio/vector/issues/260#issuecomment-565836899) note explains why.